documentation/sections/changelog.md

```diff
@@ -1,6 +1,22 @@
 ## Change Log
 
 ```
+releaseHeader('2017-02-10', '2.0.0-alpha1', '1.12.3')
+```
+*   Initial alpha release of CoffeeScript 2. The CoffeeScript compiler now outputs ES2015+ syntax whenever possible.
+*   Classes are output using ES2015 `class` and `extends` keywords.
+*   Added support for `async`/`await`.
+*   Bound (arrow) functions now output as `=>` functions.
+*   Function parameters with default values now use ES2015 default values syntax.
+*   Splat function parameters now use ES2015 spread syntax.
+*   Computed properties now use ES2015 syntax.
+*   Interpolated strings (template literals) now use ES2015 backtick syntax.
+*   Improved support for recognizing Markdown in Literate CoffeeScript files.
+*   Mixing tabs and spaces in indentation is now disallowed.
+*   Browser compiler is now minified using the Google Closure Compiler (JavaScript version).
+*   Node 7+ required for CoffeeScript 2.
```

Comment:
Yes, and probably farther back than that. Basically Node 7 added support for `async`/`await` behind a flag, so that feature requires Node 7+. I haven’t tested Node 7.0 vs 7.2; generally I’ve been working in Node latest. But `async` functions aren’t used in the CoffeeScript compiler codebase, so Node 7 or the flag aren’t required to run the compiler itself. The compiler requires full support for classes, which has been around since Node 4.

So basically, to run the compiler in Node, you need Node 4+ (possibly 6+). To run all the tests, you need 7+ and the `--harmony` flag. I haven’t done tests to see exactly which versions within those ranges work.